### PR TITLE
Show users region in top bar

### PR DESCRIPTION
**Pull request by bryangaledxw**
# Changes in this PR

We show the user's region at the top of each page

### Before
localhost_3000_ (2)

### After
localhost_3000_ (1)

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Have all changes to any dependencies been deployed to both `preprod` and
    `production` in a backwards compatible way? (This includes our API,
    infrastructure, third-party integrations and any secrets or environment variables)
- [ ] Has a required data migration been included in this change or been done in
    advance?

## Post merge checklist

Once we've merged we now need to release this through to production before
considering it done.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-temporary-accommodation-ui)
and work through the following steps:

- [ ] Has the product manager asked to provide approval for this feature?
  - [ ] Has the product manager approved?
- [ ] Manually approve release to preprod
- [ ] Verify change released to preprod
- [ ] Manually approve release to prod
- [ ] Verify change released to production

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

### CODE REVIEW

**Comment by bryangaledxw** [comment]
[Show user's region in top bar](https://trello.com/c/BYkXuaL9/667-show-users-region-in-top-bar)

File: integration_tests/tests/login.cy.ts [hunk 1]
```diff
@@ -22,10 +22,14 @@ context('SignIn', () => {
     Page.verifyOnPage(AuthSignInPage)
   })
 
-  it('User name visible in header', () => {
+  it('User name and probation region visible in header', () => {
```
Comment (patrickjfl) [inline integration_tests/tests/login.cy.ts]: Maybe it would make sense to separate this into two tests?

**Review by patrickjfl — APPROVED** [review]
LGTM, just one minor comment

NB should we be able to see the user's region in the 'Manage your details' screen? For me it's not visible there (seems to be everywhere else though)

**Comment by bryangaledxw** [comment]
@patrickjfl Ah yeah, that's more obvious in the dev environment as it's a different domain, but 'Manage your details' is outside of our control

**Pull request merged**